components/TopNav.vue

```diff
@@ -10,7 +10,7 @@
                 </svg>
                 <div class="title-text flex flex-col" data-testid="logo">
                     <div class="text-lg text-primary group-hover:text-primary-hover">
-                        Find a Doc 
+                        Find a Doc
```

Comment:
👀 looks like this one WAS unlinted 🤔

Reply:
This might be the lf vs crlf line ending problem I was taking about?